android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -90,6 +90,11 @@
       window.jsInterface.insertText(dn, s);
     }
 
+    function resetContext() {
+      var kmw = window['tavultesoft']['keymanweb'];
```

Comment:
We don't need to use this model of referencing within **keyboard.html**. The `['name']` approach is the hoopy way that Google Closures identifies names that should not be minified. **keyboard.html** will never be minified, so we can instead say `tavultesoft.keymanweb.resetContext();` and avoid `['']`land altogether here. Appreciate there may be other areas in keyboard.html that already do this, but let's not perpetuate it.